src/SwaggerWcf/Support/Mapper.cs

```diff
@@ -261,9 +261,17 @@ private static string ConcatPaths(string basePath, string pathUrl)
                     });
                 }
 
+                // parameters - body
+                int bodyParameterCount = 0;
+                foreach (ParameterInfo parameter in parameters)
```

Comment:
Maybe replace this loop with a LINQ expression?

`int bodyParameterCount = parameters.Where(p => GetInType(uriTemplate, p.Name) == InType.Body).Count;`

Reply:
Just did it.